examples/vacanza.py

```
@@ -0,0 +1,323 @@
+            f"Supported languages: {', '.join(supported_languages)}"
+        )
+        raise ValueError(message)
+
+    categories_to_generate = get_categories(
+        category=category, country_code=country_code, supported_categories=supported_categories
+    )
+
+    year_str = f"{min(years)}-{max(years)}"
+    lang_suffix = f"_{language.upper()}" if language else ""
+
+    for category in categories_to_generate:
```

Ruff (0.15.10)

[warning] Redefining argument with the local name `category`

(PLR1704)